src/Process/Listener/FileLoggerSubscriber/BaseFileLoggerSubscriber.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+namespace Infection\Process\Listener\FileLoggerSubscriber;
+
+use Infection\Config\InfectionConfig;
+use Infection\Console\LogVerbosity;
+use Infection\EventDispatcher\EventSubscriberInterface;
+use Infection\Events\MutationTestingFinished;
+use Infection\Filesystem\Filesystem;
+use Infection\Mutant\MetricsCalculator;
+
+class BaseFileLoggerSubscriber implements EventSubscriberInterface
+{
+    const TEXT_FILE = 'text';
+    const SUMMARY_FILE = 'summary';
+    const DEBUG_FILE = 'debug';
```

Comment:
Could you please add documentation for these log types?

https://infection.github.io/guide/usage.html#Configuration-settings
https://github.com/infection/site/blob/master/src/guide/usage.md